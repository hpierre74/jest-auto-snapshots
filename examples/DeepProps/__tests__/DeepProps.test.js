import snap from 'jest-auto-snapshots';
import DeepProps from '../DeepProps';

snap(DeepProps, '../DeepProps.jsx');
