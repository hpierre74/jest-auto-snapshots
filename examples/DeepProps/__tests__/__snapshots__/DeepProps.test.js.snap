// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > DeepProps Matches snapshot when array prop "users" has one item: "[object Object]": when array prop "users" has one item: "[object Object]" 1`] = `
<div>
  <div
    key="jest-auto-snapshots String Fixture"
  >
    <h1>
      jest-auto-snapshots String Fixture
       
      jest-auto-snapshots String Fixture
    </h1>
    <div
      key="1"
    >
      <h2>
        1
      </h2>
      <span>
        $
        jest-auto-snapshots String Fixture
      </span>
    </div>
  </div>
  <span>
    jest-auto-snapshots String Fixture
  </span>
</div>
`;

exports[`jest-auto-snapshots > DeepProps Matches snapshot when array prop "users" is an empty array: "[]": when array prop "users" is an empty array: "[]" 1`] = `
<div>
  <span>
    jest-auto-snapshots String Fixture
  </span>
</div>
`;

exports[`jest-auto-snapshots > DeepProps Matches snapshot when passed all props: when passed all props 1`] = `
<div>
  <div
    key="jest-auto-snapshots String Fixture"
  >
    <h1>
      jest-auto-snapshots String Fixture
       
      jest-auto-snapshots String Fixture
    </h1>
    <div
      key="1"
    >
      <h2>
        1
      </h2>
      <span>
        $
        jest-auto-snapshots String Fixture
      </span>
    </div>
  </div>
  <span>
    jest-auto-snapshots String Fixture
  </span>
</div>
`;

exports[`jest-auto-snapshots > DeepProps Matches snapshot when passed only required props: when passed only required props 1`] = `
<div>
  <div
    key="jest-auto-snapshots String Fixture"
  >
    <h1>
      jest-auto-snapshots String Fixture
       
      jest-auto-snapshots String Fixture
    </h1>
    <div
      key="1"
    >
      <h2>
        1
      </h2>
      <span>
        $
        jest-auto-snapshots String Fixture
      </span>
    </div>
  </div>
  <span />
</div>
`;
